src/test/java/com/alevel/cw3/MainTest.java

```diff
@@ -0,0 +1,137 @@
+package com.alevel.cw3;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.Scanner;
+
+import static org.junit.Assert.*;
+
+public class MainTest {
+
+    @Test
+    public void numberSum_positive() throws Exception {
+        InputOutput in = Mockito.mock(InputOutput.class);
```

Comment:
You can simplify the code if you write like this
![image](https://user-images.githubusercontent.com/18196757/142785049-09f95aa5-b897-4781-8cd2-410d37da5643.png)